workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -329,13 +329,35 @@ def _get_postphysics_stepper(
             stepper = None
         return stepper
 
-    def _open_model(self, ml_config: MachineLearningConfig, step: str):
+    def _get_radiation_stepper(
+        self, config: UserConfig, physics_namelist: Mapping[Hashable, Any]
+    ) -> Optional[Stepper]:
+        if config.radiation_scheme is not None:
+            radiation_input_generator_config = config.radiation_scheme.input_generator
+            if radiation_input_generator_config is not None:
+                radiation_input_generator: Optional[
+                    Union[PureMLStepper, Prescriber]
```

Comment:
I get this is already existing and no action required, but it feels like using a `Stepper` class to produce inputs for the radiation scheme is a bit heavy-handed. E.g. the `PureMLStepper` `__call__` does all sorts of things like computing column integrated heating/moistening which are not relevant to the radiation inputs. I wonder if there is a more lightweight common interface we could use instead of those classes.

Reply:
Agree, and don't think this is specific to the radiation stepper. There's a core of just making an ML prediction that could be refactored into a useful class. We already run into this when the prephysics ML has to specify odd things (hydrostatic flag) for making unrelated predictions (surface radiative fluxes).